games/2048.cpp/Portfile

```diff
@@ -0,0 +1,35 @@
+PortSystem              1.0
+
+name                    2048.cpp
+# no official releases
+version                 0
+# revision number is the commit count
+revision                354
```

Comment:
The `revision` field is not to be used as a `version`. Furthermore, the `revision` always starts at 0. Instead set the `version` to either the last commit date or possibly the commit count or both. For example:

`version   2020-02-01-354
`

Reply:
Personally, in these cases, I prefer just the date of the last commit.